src/buildExec.ts

```diff
@@ -287,7 +85,7 @@ const buildGeneralExec = () => {
   if (verbose) {
     args.push('-v');
   }
-  return args;
```

Comment:
seems like we should return somethign like

```javascript
{
    url,
    verbose
}
```
instead

Reply:
the reason why we're returning args here, is that it encapsulates building the args with their options names. So if we're returning url and verbose, we would have to wrap each argument with it's option name in index.ts which I don't think is the right thing to do

Reply:
Oh I see, I think we will want to do that eventually to be more clear what `args` is going to be, but can push off to another PR